### DISCUSSION TITLE
cross validation for MFNN learning from a dataset

### DISCUSSION THREAD

**Discussion by AmosJoseph** [lululxvi/deepxde#804] [General]
Excuse me, in the doc Demos of Function Approximation (Multi-fidelity learning from a dataset), how to implement  cross validation on high fidelity dataset for MFNN learning from a dataset?

Could you give a code snippe?

Many thanks!
 
Best!

**Comment by lululxvi** [comment]
- https://en.wikipedia.org/wiki/Cross-validation_(statistics)
- https://scikit-learn.org/stable/modules/cross_validation.html
- https://github.com/lululxvi/deep-learning-for-indentation
